### PR TITLE
feat(spcr-settings): unify style with the current Spotify

### PR DESCRIPTION
**Pull request by SunsetTechuila**
before:
image

after:
image

this can be called "fix" also, I guess

### CODE REVIEW

File: packages/spcr-settings/settingsSection.tsx [hunk 1]
```diff
@@ -77,7 +71,7 @@ class SettingsSection {
     if (!pluginSettingsContainer) {
       pluginSettingsContainer = document.createElement("div");
       pluginSettingsContainer.id = this.settingsId;
-      pluginSettingsContainer.className = styles.settingsContainer;
+      pluginSettingsContainer.className = "settingsContainer";
```
Comment (FlafyDev) [inline packages/spcr-settings/settingsSection.tsx]: You dropped `settings.module.css` so you removed `styles.settingsContainer`. But why replace it with `"settingsContainer"`? I don't think it does anything.

**Review by FlafyDev — CHANGES_REQUESTED** [review]
The current types try to _not_ relay on Spotify's classes as much as possible. This is because otherwise every time Spotify would change the settings' class names this plugin would break.

Also, look at the Button's styles. It finds the styles of the buttons in the settings page at runtime (see line 68) thus not having to hardcode Spotify's classes into the code ahead of time.

This PR seems to take a lot of classes from Spotify, which increases its chances to break between updates. Especially by using types with randomly generated strings such as  `Type__TypeElement-sc-goli3j-0`, and `Button-sc-y0gtbx-0` which are present in this PR.

EDIT: sorry, I meant to combine the previous comment and this comment in the same review..

**Comment by SunsetTechuila** [comment]
> The current types try to not relay on Spotify's classes as much as possible
> This PR seems to take a lot of classes from Spotify, which increases its chances to break between updates

A plugin that provides the ability to integrate your settings into Spotify's ones, should itself be integrated with Spotify as much as possible to provide seamless UX/UI between stock and mods. Because if you don't care about UI/UX unity, why use that plugin? In that case, it's more reasonable to make your own settings popup/page/panel according to your preferences and needs (and that's what most devs do).
Using your own classes (especially those generated automatically) completely destroys this paradigm, because when installing any custom theme, user will immediately see where the stock settings end and where the settings added with this plugin begin.
That's my opinion. About these classes being subject to change - these particular ones don't change very often, but I can add them to the css map if that's a worry for you.

**Comment by SunsetTechuila** [comment]
> such as `Type__TypeElement-sc-goli3j-0`, and `Button-sc-y0gtbx-0`

removed since they are doing nothing

**Comment by SunsetTechuila** [comment]
works even with `1.2.5.1006.g22820f93` (latest is `1.2.18.999.g9b38fc27`)
image

**Comment by FlafyDev** [comment]
Let's hope it'll last. thanks!

**Comment by SunsetTechuila** [comment]
don't forget to bump the version this time :)

**Pull request merged**